org.eclipse.scout.rt.uinotification/src/main/java/org/eclipse/scout/rt/api/uinotification/UiNotificationRegistry.java

```diff
@@ -352,13 +352,13 @@ protected void updateNotificationCreationTime(UiNotificationDo notification) {
     m_lastCreationTime = notification.getCreationTime();
   }
 
-  public void addListener(String topic, UiNotificationListener listener) {
+  public synchronized void addListener(String topic, UiNotificationListener listener) {
```

Comment:
Why synchronized over the instance of `UiNotificationRegistry`? Wouldn't it be more suitable to synchronize only via `m_listerners`?